CHANGELOG.md

```diff
@@ -9,6 +9,11 @@
 are made before the last `apostrophe:modulesRegistered` handler has fired.
 If you need to call Apostrophe's `find()` methods at startup,
 it is best to wait for the `@apostrophecms/doc:beforeReplicate` event.
+* Allow `@` when a piece is a template and `/@` for page templates (doc-template-library module).
```

Comment:
@falkodev like the entry below, this line was merged into a released section, where it should have been merged into UNRELEASED